Source/Fuse.Controls.Video/iOS/VideoImpl.mm

```diff
@@ -294,6 +294,10 @@ int getRotation(void * videoState)
 				auto t = track.preferredTransform;
 				auto angle = atan2((double)t.b, (double)t.a);
 				auto d = angle * 180.0 / M_PI;
+				if (d < 0)
+				{
+					d = 360.0 + d;
```

Comment:
This appears wrong. It uses 360degrees, but `d` is currently in radians `auto d = angle * 180.0 / M_PI;`

Reply:
@mortoray what? `d` is degrees, read line `296` again

Reply:
Derp! :face_with_head_bandage:  Yes, indeed. It's all degrees.